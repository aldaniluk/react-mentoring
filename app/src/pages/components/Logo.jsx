import styled from 'styled-components';

const LogoText = styled.p`
    color: #f65261;
`

const BoldText = styled.span`
    font-weight: bold;
`

function Logo() {
    return (
        <LogoText>
            <BoldText>netflix</BoldText>roulette
        </LogoText>
    )
}

export { Logo }