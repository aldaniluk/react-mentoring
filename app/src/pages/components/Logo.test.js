import React from 'react';
import { Logo } from './Logo';
import { render } from '@testing-library/react';

it('Logo testing', () => {
    const { asFragment } = render(<Logo />);

    expect(asFragment()).toMatchSnapshot();
});