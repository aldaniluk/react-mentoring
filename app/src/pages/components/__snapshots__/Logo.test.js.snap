// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo testing 1`] = `
<DocumentFragment>
  <p
    class="sc-bdfBwQ cmpjlk"
  >
    <span
      class="sc-gsTCUz jXGVpS"
    >
      netflix
    </span>
    roulette
  </p>
</DocumentFragment>
`;
